basket/settings.py

```diff
@@ -85,26 +88,27 @@ def path(*args):
 default_email_backend = "django.core.mail.backends.console.EmailBackend" if DEBUG else "django.core.mail.backends.smtp.EmailBackend"
 EMAIL_BACKEND = config("EMAIL_BACKEND", default=default_email_backend)
 EMAIL_HOST = config("EMAIL_HOST", default="localhost")
-EMAIL_PORT = config("EMAIL_PORT", default=25, cast=int)
-EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=False, cast=bool)
+EMAIL_PORT = config("EMAIL_PORT", parser=int, default="25")
+EMAIL_USE_TLS = config("EMAIL_USE_TLS", parser=bool, default="false")
 EMAIL_SUBJECT_PREFIX = config("EMAIL_SUBJECT_PREFIX", default="[basket] ")
 EMAIL_HOST_USER = config("EMAIL_HOST_USER", default="")
 EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
 
 ALLOWED_HOSTS = config(
     "ALLOWED_HOSTS",
-    default=".allizom.org, .moz.works, basket.mozmar.org, basket.mozilla.com, basket.mozilla.org",
-    cast=Csv(),
+    parser=ListOf(str, allow_empty=False),
+    default="basket-dev.allizom.org,basket.allizom.org,basket.mozilla.org",
+)
+ALLOWED_CIDR_NETS = config(
+    "ALLOWED_CIDR_NETS",
```

Comment:
In the switch from Csv() to ListOf(str), above, you take out the spaces in the comma-separated list. Do we need to also update `ALLOWED_CIDR_NETS` in the relevant env vars to not have spaces, too?

Reply:
We don't seem to set this env var in any of the environments. But I just removed the spaces b/c it's a CSV string and it matched my mental model of what that should look like. But everett removes the spaces anyway:

```python
>>> ListOf(str)('foo, bar,  baz')
['foo', 'bar', 'baz']
```